pkg/agent/manager/manager.go

```diff
@@ -10,8 +10,10 @@ import (
 	"time"
 
 	"github.com/andres-erbsen/clock"
+	"github.com/cenkalti/backoff/v3"
 	observer "github.com/imkira/go-observer"
 	"github.com/spiffe/spire/pkg/agent/client"
+	spire_backoff "github.com/spiffe/spire/pkg/agent/common/backoff"
```

Comment:
wonder if we could avoid doing named imports if we type aliased `backoff.Backoff` in the `.../agent/common/backoff` package?

Reply:
ah good call :)